salt/modules/nilrt_ip.py

```diff
@@ -600,7 +601,7 @@ def _change_connman_backlist(interface, add=True):
         parser.write(config_file)
     if add:
```

Comment:
this looks at least funny... if add remove()

Reply:
Yeah, I could refactor this